app/controllers/hosts_controller.rb

```diff
@@ -60,7 +60,7 @@ def index(title = nil)
         render :index if title && (@title = title)
       end
       format.csv do
-        @hosts = search.includes(included_associations - [:host_statuses, :token, :compute_resource])
+        @hosts = search.preload(included_associations - [:token])
```

Comment:
not directly related but took advantage of this PR for a small performance improvement.